Bumped spellcheck action to latest version (0.19.0)

jonasbn opened the pull request:
Hello,

I have just [released 0.19.0 of the GitHub Spellcheck Action](https://dev.to/jonasbn/releases-0190-of-spellcheck-github-action-a-security-release-599k) and I noticed that older versions of the Docker image had been pulled from DockerHub recently. 

I was able to locate your configuration via [SourceGraph](https://sourcegraph.com/github.com/1995parham/Learning/-/blob/.github/workflows/spell.yaml), so I am providing you with a PR proposing an update to a more contemporary version.

It should be backwards compatible even though a lot has happened, but if you experience any issue with the PR please let me know and I will comply.

Take care and stay safe,

jonasbn

1995parham commented:
@jonasbn thanks for your time and attention, I very much appreciate it. Sure, I will be glad if you can create a PR for Dependabot.

jonasbn commented:
Hi @1995parham 

I have added a proposal for a [Dependabot](https://github.com/dependabot) configuration aimed at GitHub Actions.

This is the configuration I use myself, it is **only** aimed at GitHub Actions, since I do not know anything about your project in general.

The PR enables use of Dependabot, which will scheduled intervals will create PRs for you to review and possibly merge. Since you only have one action at this time, you will not receive many PRs from Dependabot. I must admit I like to keep the schedule on a weekly basis, but you can alter this to fit your schedule.

This is a very basic configuration, which works well for me, more [documentation](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot) is available (also linked in the code as a comment, something I find very convenient, when it is not your core project and not something you work on every day..

I have just release 0.20.0 of the Spellcheck GitHub Action, so if you enable Dependabot, you should receive a PR shortly.

Please use the PR as you see fit, change it, adjust it or discard it.

Take care and stay safe

jonasbn commented:
This is the result of a local execution:

```text
pyspelling -c spellcheck.yaml
Misspelled words:
<text> english/present-questions.txt
--------------------------------------------------------------------------------
Parham
kils
kish
whos
--------------------------------------------------------------------------------

Misspelled words:
<text> english/first-condition.txt
--------------------------------------------------------------------------------
homeworks
wil
--------------------------------------------------------------------------------

!!!Spelling check failed!!!
```

I have corrected some of these and added a few to the local dictionary (`.wordlist.txt`)
